Dockerfile_Flatzingo

```diff
@@ -52,7 +53,8 @@ RUN apt-get update -y && apt-get install software-properties-common -y && \
 COPY --from=builder /install /install
 
 # Copy flatzingo's executable from the previous stage across
-COPY --from=builder /flatzingo/*.lp /entry_data/
+RUN mkdir -p /entry_data/encodings
+COPY --from=builder /flatzingo/encodings/* /entry_data/encodings
```

Comment:
TODO: I'll have to check if the docker still runs

Reply:
I already checked and it did, as I said, this wont really work untill it is in master.... or you can just change the line in the docker that is commented so that it pulls from my fork instead